GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+# Prometheus Governance 2.0
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. This governance is an open, living document, and will continue to evolve as the community and project change.
```

Comment:
The document keeps referring to "the project" instead of directly mentioning Prometheus. It feels like Fight Club (Rule 1: Don't talk about Fight Club).

It would be helpful to add an introduction that clearly states what the project is.

Reply:
That's probably because we copied this sentence and some more from a CNCF template. We should update this. Good catch!